<launch>
    <param name="lower_h" type="int" value="150" />
    <param name="lower_s" type="int" value="100" />
    <param name="lower_v" type="int" value="20" />
    <param name="upper_h" type="int" value="180" />
    <param name="upper_s" type="int" value="255" />
    <param name="upper_v" type="int" value="255" />
    <param name="cam_rate" type="int" value="4" />
    <node pkg="camera" name="camera_node" type="cam_test.py" output="screen"/>
</launch>